core.multiview/src/org/netbeans/core/multiview/SplitAction.java

```diff
@@ -112,13 +111,17 @@ static boolean isSplitingEnabled() {
 	    if (tc != null) {
 		setEnabled(true);
 		if (tc instanceof Splitable && ((Splitable)tc).canSplit()) {
-		    JMenuItem item = new JMenuItem(new SplitDocumentAction(tc, JSplitPane.VERTICAL_SPLIT));
+
+		    JMenuItem item = new JMenuItem(new SplitDocumentVerticallyAction());
```

Comment:
... not really certain we need to use SplitDocumentVerticallyAction here, the existing SplitDocumentAction works just as well